.github/workflows/lint.yml

```diff
@@ -88,6 +72,9 @@ jobs:
 
       - name: Install module (and test requirements)
         run: pip install -e .[test]
+      
```

Comment:
Do we need to install the sample dependencies?

Reply:
The linter would raise 'import not found' error in the sample code if we don't.